avocado/core/test.py

```diff
@@ -373,22 +375,34 @@ def _start_logging(self):
         stream_fmt = '%(message)s'
         stream_formatter = logging.Formatter(fmt=stream_fmt)
 
-        self._register_log_file_handler(logging.getLogger("avocado.test.stdout"),
-                                        stream_formatter,
-                                        self._stdout_file)
-        self._register_log_file_handler(logging.getLogger("avocado.test.stderr"),
-                                        stream_formatter,
-                                        self._stderr_file)
-        self._ssh_fh = self._register_log_file_handler(logging.getLogger('paramiko'),
-                                                       formatter,
-                                                       self._ssh_logfile)
+        self._register_log_file_handler(
+                                      logging.getLogger("avocado.test.stdout"),
```

Comment:
Hmm, I don't really like this style, nor I saw it before. I'm used to either using shorter variable names, assign the necessary variable to another variable, wrap it and start with one tab or keep the long line. But I never saw decreasing the length of one tab in order to fit in. Are you sure this was intended?